spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/servlet/view/MustacheView.java

```diff
@@ -66,7 +66,7 @@ public void setCharset(String charset) {
 	}
 
 	@Override
-	public boolean checkResource(Locale locale) throws Exception {
```

Comment:
This is a breaking change as the method is public API. Please revert.

Reply:
Thank you for the review. I’ve updated the code to address the points you mentioned. Please have another look when you have a chance.